exchanges/gateio/gateio_wrapper.go

```diff
@@ -1154,6 +1154,8 @@ func (g *Gateio) CancelBatchOrders(ctx context.Context, o []order.Cancel) (order
 	var response order.CancelBatchResponse
 	if len(o) == 0 {
 		return response, errors.New("no cancel order passed")
+	} else if len(o) > 20 {
+		return response, fmt.Errorf("%w maximum order size,20 cancel orders, exceeded", errMultipleOrders)
 	}
```

Comment:
You can run a for loop to cancel all requested orders here. Instead of erroring.